mithril-signer/Cargo.toml

```diff
@@ -23,9 +23,11 @@ config = "0.14.0"
 hex = "0.4.3"
 mithril-common = { path = "../mithril-common", features = ["full"] }
 mithril-doc = { path = "../internal/mithril-doc" }
+mithril-metric = { path = "../internal/mithril-metric" }
 mithril-persistence = { path = "../internal/mithril-persistence" }
 openssl = { version = "0.10.66", features = ["vendored"], optional = true }
 openssl-probe = { version = "0.1.5", optional = true }
+paste = "1.0.15"
```

Comment:
I think this dependency can be removed from `mithril-signer` since it doesn't use it.

Reply:
It need it to use macro.

Reply:
Even if you use fully qualified path for paste inside the `build_metrics_service` ?